hack/scale_out_poc/config_haproxy/cfg_generator/util.go

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
An updated version of this file has already been moved to master branch in #929. This file should be removed.

Reply:
deleted